src/utils/ruleCalls.js

```diff
@@ -1,10 +1,19 @@
 import axios from "axios";
 
 export const listRules = async () => {
-  const response = await axios.get(process.env.API_URL);
+  const response = await axios.get(`${process.env.API_URL  }/bounce_rules`);
   if (response.status === 200) {
-    return await response;
-  } else {
-    throw new Error("Error retrieving all rules");
+    return response;
   }
+  throw new Error("Error retrieving all rules");
+};
+
+export const getRule = async ruleId => {
+  const response = axios.get(`${process.env.API_URL}/${ruleId}`).catch(err => {
+    throw new Error(err);
+  });
+  if (response.status === 200) {
```

Comment:
I think you can also do `if (response.ok)` here. no need to change, more of an fyi